tf2onnx/schemas.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.schema
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict, OrderedDict
+from onnx import defs
+
+
+ONNX_DOMAIN = ""
+
+
+class ONNXOpSchema(object):
+    """Wrapper for ONNX schema."""
+
+    def __init__(self, name, domain, since_version, attributes):
+        """Create a ONNX schema
+        Args:
+            name (str): op name
+            attributes (List[str]): valid attributes
+            domain (str): default value "" means it's ONNX domain
+            since_version (int): opset version, default is 1
+        """
+        self._name = name
+        self._domain = domain
+        self._attributes = attributes
+        self._since_version = since_version
+
+    @property
+    def attributes(self):
+        return self._attributes
+
```

Comment:
better to add a static method, something like ```parse```, which parse 1 of the schema that get from ```defs.get_all_schemas_with_history()``` into an ONNXOpSchema and return.

keep the logic in this class, so in case we need parse other fields, e.g. type constraints, just change here.